doc/content/devices/models/browan-tbms100/_index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Browan TBMS100"
+description: ""
+weight: 
+---
+
+{{< figure src="tbms100.png" alt="Generic Node Sensor Edition" class="float plain" >}}
+
+The Browan [TBMS100](https://www.browan.com/product/motion-sensor-pir/detail) is a PIR motion sensor designed to detect occupancy or movement in homes or buildings for security and efficient facility management. It helps optimize the utilization of spaces like desks, conference rooms, and bathrooms.
+
+<!--more-->
+
+# Onboarding to {{% tts %}}
+
+The provisioning information, such as **JoinEUI** and **DevEUI**, can be found on the back of the device. 
+
+{{< figure src="tbms100-back.png" alt="TBMS100 provisioning information" >}}
+
+{{< note "The sensitive information, the **AppKey**, is provided within an Excel sheet via email to the person who purchased it." />}}
+
+The TBMS100 doesn't support onboarding with a QR code. You must proceed directly to the options [**Select the end device in the LoRaWAN Device Repository**]({{< ref "#using-the-lorawan-device-repository" >}}) or [**Enter end device specifics manually**]({{< ref "#manually" >}}).
+
+## Using the LoRaWAN Device Repository
+
+The [LoRaWAN® Device repository](https://github.com/TheThingsNetwork/lorawan-devices) comprises over 600 end-device profiles, including the Browan TBMS100, which requires less information for registration with {{% tts %}}.
+
+In the **End device type** section, under **Input method**, select **Select the end device in the LoRaWAN Device Repository** option. The [Onboarding devices without QR codes]({{< ref "/devices/adding-devices/#onboarding-devices-without-qr-codes" >}}) section of the [Adding Devices]({{< ref "/devices/adding-devices/" >}}) page explains all the configurations you need to provide. However, the following are the TBMS100 specific configurations you want to include:
```

Comment:
```suggestion
In the **End device type** section, under **Input method**, select **Select the end device in the LoRaWAN Device Repository** option. The [Onboarding devices without QR codes]({{< ref "/devices/adding-devices/#onboarding-devices-without-qr-codes" >}}) section of the [Adding Devices]({{< ref "/devices/adding-devices/" >}}) guide explains this process in detail. Following details are TBMS100-specific:
```

Reply:
Please don't use "all the configurations" in general. We are talking about one end device configuration at a time, with multiple parameters being set